src/renderer/utils/tootParser.ts

```diff
@@ -106,3 +108,12 @@ export function parsePleromaAccount(accountURL: string): Account | null {
     url: accountURL
   }
 }
+
+export function isRTL(content: string): boolean{
+  //based on the first character check if Hebrew or arab
```

Comment:
For mentions, the first character will always be @, so posts will be parsed as LTR even if they're RTL. The func should trim mentions from the beginning of the content before applying the pattern.

Reply:
> I can't read/write any RTL languages, so could you please attach images of the final rendered result?

I've used a solution based on the first character, though its not optimal (see notes below)
It still predicts better than the `OriginalMessage.value.language` element

![image](https://user-images.githubusercontent.com/86617266/202232622-378e0ba4-9066-40d6-b402-5dc89be7c652.png)

Note the difference between the Hebrew/English toots (English one remains unchanged)


> For mentions, the first character will always be @, so posts will be parsed as LTR even if they're RTL. The func should trim mentions from the beginning of the content before applying the pattern.

* There is also the case of using emojis/other characters at the beginning.
* In 'tusky' I saw that they use a per-line solution for the text and not for the whole block.
I'm not very proficient in js/ts, maybe you can help in parsing the text itself and adding the alignment were needed instead of globally for the whole wrapping div.
* I now notice that end of sentence punctuation appear at the beginning for some reason - probably need alignment in the containing <p> tags inside the returned message

Reply:
Hey, @yperetz , I created a PR in your fork to change the logic for isRTL to strip out non-letter characters. You can merge my PR in, then update your PR here.